userappwithbroker/build.gradle

```diff
@@ -60,12 +60,14 @@ android {
 
     buildTypes {
         debug {
+            minifyEnabled false
```

Comment:
Should this be true? (so that we can catch potential issue before merging in PR)

Reply:
I'm OK with this.  That's how I tested it.